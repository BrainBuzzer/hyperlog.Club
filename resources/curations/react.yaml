name: ReactJS
description: React is a JavaScript library for building user interfaces. It is maintained by Facebook and a community of individual developers and companies.
topics:
  - name: React Basics
    lessons:
      - name: Official ReactJS Tutorial
        link: https://reactjs.org/tutorial/tutorial.html?ref=hyperlog.club
        description: This is official ReactJS tutorial which should work as a good guide to get you up and running with certain aspects of ReactJS.
        difficulty: green
      - name: React Tutorial by Tania Rascia
        link: https://www.taniarascia.com/getting-started-with-react/?ref=hyperlog.club
        description: This tutorial by Tania Rascia gets you from zero to introducing all the concepts in the React ecosystem.
        difficulty: red
  - name: Intermediate React
    lessons:
      - name: Official ReactJS Tutorial
        link: https://reactjs.org/tutorial/tutorial.html?ref=hyperlog.club
        description: This is official ReactJS tutorial which should work as a good guide to get you up and running with certain aspects of ReactJS.
        difficulty: green
      - name: React Tutorial by Tania Rascia
        link: https://www.taniarascia.com/getting-started-with-react/?ref=hyperlog.club
        description: This tutorial by Tania Rascia gets you from zero to introducing all the concepts in the React ecosystem.
        difficulty: green
  - name: Advanced React
    lessons:
      - name: Official ReactJS Tutorial
        link: https://reactjs.org/tutorial/tutorial.html?ref=hyperlog.club
        description: This is official ReactJS tutorial which should work as a good guide to get you up and running with certain aspects of ReactJS.
        difficulty: green
      - name: React Tutorial by Tania Rascia
        link: https://www.taniarascia.com/getting-started-with-react/?ref=hyperlog.club
        description: This tutorial by Tania Rascia gets you from zero to introducing all the concepts in the React ecosystem.
        difficulty: green
resources:
  - name: Awesome React
    link: https://github.com/enaqx/awesome-react?ref=hyperlog.club
    description: A collection of awesome things regarding React Ecosystem.
