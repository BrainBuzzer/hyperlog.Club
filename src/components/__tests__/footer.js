import React from "react"
import { render } from "@testing-library/react"

import Footer from "../footer"

describe("Footer", () => {
  it("renders correctly", () => {
    const { asFragment }= render(<Footer />)
    expect(asFragment()).toMatchSnapshot()
  })
})
