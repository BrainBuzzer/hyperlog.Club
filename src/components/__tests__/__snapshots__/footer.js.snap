// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<DocumentFragment>
  <footer
    css="[object Object]"
  >
    Â© 2019, Hyperlog
  </footer>
</DocumentFragment>
`;
